.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
Please add newline at end.

Reply:
It's there 😉 , I think GitHub diff might not be showing it that clearly. When there is no space, you usually see the "cancel" like symbol in the diff

Reply:
Do you want me to add an EditorConfig in a separate PR to enforce that (except for the test files)?

Reply:
No thanks - I'm not sold on editor config.